config/samples/resources/bigqueryconnectionconnection/cloudsql-connection/bigqueryconnection_v1alpha1_bigqueryconnectionconnection.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: bigqueryconnection.cnrm.cloud.google.com/v1beta1
+kind: BigQueryConnectionConnection
+metadata:
+  name: bigqueryconnectionconnection-sample
+spec:
+  location: us-central1
+  projectRef:
+    external: ${projectId}
+  cloudSQL:
+    instanceRef:
+      name: sqlinstance-dep
+    database: sqldatabase-dep
```

Comment:
According the naming convention for samples [1], this should be named `bigqueryconnectionconnection-dep`.

[1] https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.Samples.md#dependency-naming-convention